docs/docusaurus/docs/cloud/about_gx.md

```diff
@@ -52,6 +52,10 @@ The GX Agent is an intermediary between GX Cloud and your organization's data st
 
 The GX Agent is typically deployed in your organization's deployment environment, for example, in a development, staging, or production cloud services environment. The GX Agent serves all GX Cloud users within your organization. It can be run as part of your development or production workflows. See [Connect GX Cloud](./connect/connect_lp.md) for setup instructions.
 
+### GX Agent versioning
```

Comment:
What do we think about moving this to between Roles and Responsibilities and Supported Browsers? Just thinking of the newspaper article model, this more of a detail that can come later than the more critical bits e.g. deployment patterns. 
But I totally get leaving this here with the description of the agent so I won't block on this point, just wanted to consider.

Reply:
The GX Agent version is related to the GX Agent content that precedes it. So, this is the best location for the information.